django_app/redbox_app/redbox_core/tasks/README.md

```diff
@@ -0,0 +1,32 @@
+# Tasks with Django-Q2
+
+## Django-Q2
+
+We're using [Django-Q2](https://django-q2.readthedocs.io/) for handling scheduled tasks.
+This is a fork of the now-inactive Django-Q project.
+
+This was chosen because it allows us to schedule, run and monitor tasks with limited additional infrastructure, as well as being a project that is relatively popular and well-maintained.
+
+The selected message broker is the Postgres database, which enables us to schedule, manage and monitor tasks from Django Admin.
+
+In the longer term, if we end up with many more tasks, we might want to go for a different broker or even task runner, but this will involve setting up more infrastructure.
+
+## Scheduling tasks
+
+Tasks can be scheduled, managed and monitored from Django Admin at: `/admin/django_q/`.
+
+Task functions can be from anywhere in the django codebase; to organise them, there is a `tasks` directory inside `redbox_core`.
+
+To schedule a new task, visit `/admin/django_q/schedule/` and 'Add Scheduled Task'.
```

Comment:
We could write a migration to pre-populate this task.

Reply:
ooh good idea